crypto/config.go

```diff
@@ -0,0 +1,45 @@
+package crypto
```

Comment:
I think `crypto` is not common for all services, recommend to keep it in `conflux-mall` repo.

Anyway, `HMAC` based utilites is preferred in this repo, because many services need to configure an account to interact with blockchain.

Reply:
crypto package removed